docker-compose.yml

```diff
@@ -14,4 +14,6 @@ services:
             context: .
             dockerfile: './apps/api/Dockerfile'
         ports:
-            - 3333:3333
\ No newline at end of file
+            - 3333:3333
```

Comment:
You don't need to expose the port 27017 anymore to the external network.

Reply:
I understand, but this is for using mongo tools. Like mongo atlas and see what's going on in the database. Should I steel remove it?

Reply:
Do we have such services included at this moment? If not, do you think it would be more secure to remove it? 

We can always choose to expose it later when we connect any service that requires access.

Reply:
Got it, I'll remove it